addons/web_editor/static/src/js/editor/odoo-editor/test/spec/color.test.js

```diff
@@ -1,9 +1,9 @@
 import { BasicEditor, testEditor, unformat } from '../utils.js';
 import { rgbToHex } from '../../src/utils/utils.js';
 
-const setColor = (color, mode) => {
+const setColor = (color, mode, opacity) => {
     return async editor => {
-        await editor.execCommand('applyColor', color, mode);
+        await editor.execCommand('applyColor', color, mode, undefined, opacity);
```

Comment:
@adsa-odoo try this `await editor.execCommand('applyColor', color, mode, _, opacity)`  instead of undefined if we want to skip passing any value that argument, @dmo-odoo what is your take on this.